collect_app/src/main/java/org/odk/collect/android/provider/InstanceProviderAPI.java

```diff
@@ -33,6 +33,7 @@ private InstanceProviderAPI() {
     public static final String STATUS_INCOMPLETE = "incomplete";
     public static final String STATUS_COMPLETE = "complete";
     public static final String STATUS_SUBMITTED = "submitted";
+    public static final String STATUS_SUBMITTED_SMS = "submitted_sms";
```

Comment:
If you are adding a new column you have to increase databaseVersion, implement a method that performs upgrading etc. here is an example of how it should be done (for forms.db but it looks the same) https://github.com/grzesiek2010/collect/commit/4cc126419422da1da4df786d990af1e0ecf2cdf8

Reply:
@grzesiek2010 The idea is to add another status value, not another column. So now the status column can have values complete, submitted or submitted_sms. That's why the tests you highlighted before use `or` and not `and`. 

Do you see a problem with that approach? It seems like it lets us express everything we need but if you're seeing an issue, please do share!!

Reply:
Ah ok Got it, so the approach is ok but still, I think it should be `and` not `or` because if we want to get unsent forms we need to search for `!=STATUS_SUBMITTED and != STATUS_SUBMITTED_SMS`.

Reply:
Yes, just saw the error good sir. Thank you.